config/7.11.1/common.yaml

```
@@ -941,6 +941,44 @@ servers:
                       #[[end]]
                     #[[end]]
                     ## AMQ Streams END
+                    ## JBPM Kafka Emitter BEGIN
+                      #[[if .KafkaJbpmEventEmitters]]
+                    - name: KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ENABLED
+                      value: "true"
+                      #[[if .KafkaJbpmEventEmitters.BootstrapServers]]
+                    - name: KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_BOOTSTRAP_SERVERS
+                      value: "[[.KafkaJbpmEventEmitters.BootstrapServers]]"
+                      #[[end]]
+                      #[[if .KafkaJbpmEventEmitters.Acks]]
+                    - name: KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ACKS
+                      value: "[[.KafkaJbpmEventEmitters.Acks]]"
+                      #[[end]]
+                      #[[if .KafkaJbpmEventEmitters.ClientID]]
+                    - name: KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_CLIENT_ID
+                      value: "[[.KafkaJbpmEventEmitters.ClientID]]"
+                      #[[end]]
+                      #[[if .KafkaJbpmEventEmitters.MaxBlockMs]]
+                    - name: KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_MAX_BLOCK_MS
+                      value: "[[.KafkaJbpmEventEmitters.MaxBlockMs]]"
+                      #[[end]]
+                      #[[if .KafkaJbpmEventEmitters.DateFormat]]
+                    - name: KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_DATE_FORMAT
+                      value: "[[.KafkaJbpmEventEmitters.DateFormat]]"
+                      #[[end]]
+                     #[[if .KafkaJbpmEventEmitters.ProcessesTopicName]]
+                    - name: KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_PROCESSES_TOPIC_NAME
+                      value: "[[.KafkaJbpmEventEmitters.ProcessesTopicName]]"
+                      #[[end]]
+                      #[[if .KafkaJbpmEventEmitters.TaskTopicName]]
+                    - name: KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_TASK_TOPIC_NAME
+                      value: "[[.KafkaJbpmEventEmitters.TaskTopicName]]"
+                      #[[end]]
+                      #[[if .KafkaJbpmEventEmitters.CasesTopicName]]
+                    - name: KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_CASES_TOPIC_NAME
+                      value: "[[.KafkaJbpmEventEmitters.CasesTopicName]]"
```

Use plural in all topic names
```suggestion
                    - name: KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_PROCESSES_TOPIC_NAME
                      value: "[[.KafkaJbpmEventEmitters.ProcessesTopicName]]"
                      #[[end]]
                      #[[if .KafkaJbpmEventEmitters.TasksTopicName]]
                    - name: KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_TASKS_TOPIC_NAME
                      value: "[[.KafkaJbpmEventEmitters.TasksTopicName]]"
                      #[[end]]
                      #[[if .KafkaJbpmEventEmitters.CasesTopicName]]
                    - name: KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_CASES_TOPIC_NAME
                      value: "[[.KafkaJbpmEventEmitters.CasesTopicName]]"
```

---

ok